moveit_core/collision_detection_bullet/include/moveit/collision_detection_bullet/tesseract/bullet_cast_simple_manager.h

```diff
@@ -0,0 +1,124 @@
+/**
+ * @file bullet_cast_simple_manager.h
+ * @brief Tesseract ROS Bullet cast(continuous) simple collision manager.
+ *
+ * @author Levi Armstrong
+ * @date Dec 18, 2017
+ * @version TODO
+ * @bug No known bugs
+ *
+ * @copyright Copyright (c) 2017, Southwest Research Institute
+ *
+ * @par License
+ * Software License Agreement (BSD-2-Clause)
+ * @par
+ * All rights reserved.
+ * @par
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ * @par
+ *  * Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ *  * Redistributions in binary form must reproduce the above
+ *    copyright notice, this list of conditions and the following
+ *    disclaimer in the documentation and/or other materials provided
+ *    with the distribution.
+ * @par
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#ifndef TESSERACT_COLLISION_BULLET_CAST_SIMPLE_MANAGERS_H
+#define TESSERACT_COLLISION_BULLET_CAST_SIMPLE_MANAGERS_H
+
+#include <moveit/collision_detection_bullet/tesseract/bullet_utils.h>
+#include <moveit/collision_detection_bullet/tesseract/continuous_contact_manager_base.h>
+
+namespace tesseract
+{
+namespace tesseract_bullet
+{
+/** @brief A simple implementaiton of a tesseract manager which does not use BHV */
+class BulletCastSimpleManager : public ContinuousContactManagerBase
```

Comment:
Once BVH is working sufficiently, we should remove the simple manager classes to simplify things a bit.

Reply:
For the CCD, I didn't even try to port first the BulletCastSimpleManager and instead went directly for the BullectCastBVHManager as I think its implementation is cleaner (for example the looping over all collision objects is left to Bullet instead of doing the loop in tesseract). This means that BVH managers are working for both discrete and continuous cases. Do you think that I can remove the simple managers already?

Reply:
Cool! Yes, if they're both working, I'd remove the simple managers.